src/loaders/ObjectLoader.js

```diff
@@ -564,13 +564,55 @@ Object.assign( ObjectLoader.prototype, {
 				}
 
 				if ( data.format !== undefined ) texture.format = data.format;
+				if ( data.type !== undefined ) texture.type = data.type;
+				if ( data.encoding !== undefined ) texture.encoding = data.encoding;
 
 				if ( data.minFilter !== undefined ) texture.minFilter = parseConstant( data.minFilter, TEXTURE_FILTER );
 				if ( data.magFilter !== undefined ) texture.magFilter = parseConstant( data.magFilter, TEXTURE_FILTER );
 				if ( data.anisotropy !== undefined ) texture.anisotropy = data.anisotropy;
 
+				if ( data.mipmaps !== undefined ) {
+
+					var TYPED_ARRAYS = {
+						Int8Array: Int8Array,
+						Uint8Array: Uint8Array,
+						// Workaround for IE11 pre KB2929437. See #11440
+						Uint8ClampedArray: typeof Uint8ClampedArray !== 'undefined' ? Uint8ClampedArray : Uint8Array,
+						Int16Array: Int16Array,
+						Uint16Array: Uint16Array,
+						Int32Array: Int32Array,
+						Uint32Array: Uint32Array,
+						Float32Array: Float32Array,
+						Float64Array: Float64Array
+					};
```

Comment:
I copied `TYPED_ARRAYS` from `BufferGeometryLoader`. Wondering if we can share this table across modules...